### ISSUE TITLE
Crash after drink night salts 

### ISSUE THREAD

**Issue by sybsysj** [DaFuqs/Spectrum#643]
**Describe the bug**
It crashes shortly after drinking night salts.

**To Reproduce**
Drink night salts ，wait a few seconds，crash

**Minecraft version**
1.20.1

**Mod version**
1.8.3

**Crash Report**
[crash.zip](https://github.com/user-attachments/files/18236600/crash.zip)

**Comment by Gronglegrowth** [comment]
To my understanding - I investigated this a little bit - this is a Connector exclusive crash. I _guess_ Forge modifies a vanilla method that Spectrum does not anticipate should be. I did not check if that's true, but nothing in my examination of vanilla 1.20.1's code stuck out to me as to be able to cause this.

See [this Spectrumcord conversation](https://discord.com/channels/906264467636109333/923661883452588053/1320202288567947294) where I posted a bit of details, before.

**Comment by DaFuqs** [comment]
Forge modified the block states of beds, it seems.
That transformation is something that would be needed to be handled in connector for best compatibility.

**Comment by Gronglegrowth** [comment]
No, the issue is on Connector's end actually. Forge is not in the wrong here. This was fixed in [unilock's fork](https://github.com/unilock/ForgifiedFabricAPI/actions/runs/12719528838) of Forgified Fabric API.
